src/wallet-user/trust-protection/RecoveryProcess.css

```diff
@@ -31,3 +31,31 @@
 .RecoveryProcess .NetworkWarning+.alert-activated {
     margin-top: 30px;
 }
+
+.RecoveryProcess .tab-content {
+    padding: 0;
+}
+
+.RecoveryProcess .content {
+    padding: 20px;
+}
+
+.RecoveryProcess .Table .body {
+    overflow-y: auto;
+    height: calc(100vh - 575px);
+}
+
+.RecoveryProcess .recovery-process-footer {
+    background: linear-gradient(to top, #e95b5b7f, #152665 50%);
```

Comment:
Even simpler, nice!

Reply:
For the record, the 50% opacity is depicted by appending `7f` at the end of the first colour code (thanks to your explanation), while the 50% located after the  colour code `#152665` indicates that the gradient is finished at the half of the 230 px height rectangle.